android/src/main/java/com/github/reactNativeMPAndroidChart/charts/LineChartManager.java

```diff
@@ -92,4 +95,21 @@ void dataSetConfig(IDataSet<Entry> dataSet, ReadableMap config) {
             lineDataSet.enableDashedLine(lineLength, spaceLength, phase);
         }
     }
+
+    @Override
+    Entry createEntry(ReadableArray values, int index) {
```

Comment:
only LineChartManager and scatterChartManager share the default implementation. so I extract all createEntry to standalone DataSetUtils.